### ISSUE TITLE
[Realtime with mercure for flutter] There are somme implementation for flutter app ?

### ISSUE THREAD

**Issue by Dahkenangnon** [dunglas/mercure#224]
Hi I'm asked for any implementation of mercure protocole for flutter application in order to have the real time data update.

PS: I want to use symfony for backend.

Thank for any response...

**Comment by JoergSommer** [comment]
Hi!

I installed the mercure server by docker and with http it works fine. Now I have to change to https and get an SSL error with both, publishing and subscribing.

I tought the certificate is generated automatically with letsencrypt, but it seems to do not working.

So I tried to give working certificates to the server with CERT_FILE and KEY_FILE like:

docker run -itd --name mercure \
    -e JWT_KEY='<my secret key>' \
    -e ACME_HOSTS='<my domain>' \
    -e CERT_FILE="/etc/letsencrypt/live/<my domain>/cert.pem" \
    -e KEY_FILE="/etc/letsencrypt/live/<my domain>/privkey.pem" \
    -p 8080:80 -p 4430:443 \
    <docker-image>

But I get an error:
FATA[0000] open /etc/letsencrypt/live/<my domain>/cert.pem: no such file or directory

But the file is there!

I guess I have to copy the certificates to the docker image but this is not so easy because there is no shell etc.

Thx for help!
